template-notifications.php

```diff
@@ -73,10 +73,10 @@ class="button"><?php esc_html_e( 'Unread', 'disciple_tools' )?>
                         <div class="grid-x grid-margin-x grid-margin-y">
                             <div class="small-12 medium-6 medium-offset-3 cell center">
                                 <a id="next-all" onclick="get_notifications( true, false )" style="display:none;">
-                                    <?php esc_html_e( 'load more notifications', 'disciple_tools' )?>
+                                    <?php esc_html_e( 'Load more notifications', 'disciple_tools' )?>
```

Comment:
Either all or none?

Reply:
I understood this as being a sentence and not a heading, so sentences start with a capital of the first word.

Reply:
Let's try to have consistency. Right now on that page we have:
![image](https://user-images.githubusercontent.com/24901539/80018889-a1bf0000-84ce-11ea-87f6-087ca2f0519c.png)
![image](https://user-images.githubusercontent.com/24901539/80018901-a71c4a80-84ce-11ea-9f0d-24e7cb06c18d.png)

Reply:
Ok - I have set the text as "Load more notifications" and "Mark all as read"